README.md

```diff
@@ -1,61 +1,66 @@
 [![Build Status](https://travis-ci.org/daostack/daostack.svg?branch=master)](https://travis-ci.org/daostack/daostack)
 [![NPM Package](https://img.shields.io/npm/v/daostack-arc.svg?style=flat-square)](https://www.npmjs.org/package/daostack-arc)
+# DAOstack Arc
 
+Arc is the lower layer of the DAOStack. It consists of several smart contracts deployed on the Ethereum blockchain that define the basic building blocks and standard components that can be used to implement any DAO.
 
-# DAOStack ARC
+## Resources
 
-DAOStack ARC is the lower layer on the DAOStack stack.
-It provides a set of smart contracts which can be used as a building blocks for DAOs.
-Using this set of smart contracts one can define and setup organization with high level of flexibility by choosing the organization's components and schemes.
+- [Reference Documentation](docs/ref/README.md)
```

Comment:
Is `Reference documentation` basically automated documentation?
If so, maybe we should rename it for easier navigation.

Reply:
it points to the generated docs which are the combination of automatically generated docs and manual `headers`